app/config/roles.js

```diff
@@ -39,8 +39,7 @@ groupRoles.set(KANSELARIJ, [
   'manage-newsletter-infos',
   'manage-decision-publications',
   'manage-document-publications',
-  'manage-themis-publications',
-  'manage-settings',
+  'manage-themis-publications'
```

Comment:
At least in the test we seem to expect Kanselarij group to be able to access the settings. If this isn't actually the case, I'll try to rework the tests that depend on that.

Reply:
Sorry, will re-add that one. Out of scope of this story :blush: